executor_manager/executors/docker/utils.py

```
@@ -568,5 +568,35 @@ def get_container_status(container_name: str) -> dict:
+    Returns:
+        task_id string if found, None otherwise
+    """
+    try:
+        cmd = [
+            "docker",
+            "inspect",
+            "--format",
+            '{{index .Config.Labels "task_id"}}',
+            container_name,
+        ]
+        result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)